.github/workflows/ci.yml

```diff
@@ -19,6 +22,17 @@ jobs:
         test_flags: ["", "--no-default-features", "--all-features"]
     timeout-minutes: 8
     steps:
+      - name: Install sudo package
+        run: apt update && apt install sudo
+      - name: Install dependencies
+        run: |
+          sudo apt-get update
+          sudo apt-get -y install git default-jre curl clang pkg-config libssl-dev
```

Comment:
I know the emulators are written in java so i understand needing the jre, and ssl for tls. What requires curl and clang?

Reply:
oh is it the rust actions?

Reply:
Yes, I think you might be right, though I'm not entirely sure. I believe Xavier started this off with as few dependencies as possible & added them as necessary, so I'm sure they're required, but we can also try removing them & seeing what breaks if you're interested.

Reply:
My only consideration was if it would meaningfully add to CI time. It looks like the tests run in about the same time as other PRs though, so it's probably negligible